test/runtime_test.dart

```diff
@@ -50,26 +48,52 @@ void main() {
   test.group('All example programs run without errors', () {
     Directory directory = Directory('./examples/');
     directory.listSync().forEach((file) {
-      test.test('${file.path}', () async {
-        var process =
-            await TestProcess.start('dart', ['./lib/birb.dart', file.path]);
+      List dirSplit = file.path.split('.');
```

Comment:
This is hacky and doesn't feel right. You can simply check if the returned object is a File instance or Directory like this
`file is File` or `file is Directory`.